### PR TITLE
Add subscription support

### PR DESCRIPTION
**Pull request by matthewtanner91**

### CODE REVIEW

File: moesifmiddleware.go [hunk 1]
```diff
@@ -231,6 +231,41 @@ func UpdateCompaniesBatch(companies []*models.CompanyModel, configurationOption
...
+		moesifClient(configurationOption)
+	}
+
+	// Add event to the queue
+	errUpdateSubscriptionsBatch := apiClient.QueueSubscriptions(subscriptions)
+	// Log the message
+	if errUpdateSubscriptionsBatch != nil {
+		log.Fatalf("Error while updating subscriptions in batch: %s.\n", errUpdateSubscriptionsBatch.Error())
+	} else {
+		log.Println("Updated subscriptions successfully added to the queue")
```
Comment (praves77) [inline moesifmiddleware.go:265]: nitpic: generally, we log success / info only if `debug` is enabled.

**Review by praves77 — APPROVED** [review]
lftm : just a nitpic to avoid flooding of log.

File: moesifmiddleware.go (hunk 1 above)
Comment (matthewtanner91) [reply moesifmiddleware.go:265]: @praves77 followed the User and Company code so we might need to actually add in the debug flag (as I don't currently see it in similar methods).

Comment (praves77) [reply moesifmiddleware.go:265]: @matthewtanner91 no worries. Thanks.

**Pull request merged**